wiki/en/Running-a-Server.md

```diff
@@ -100,12 +89,14 @@ If you want to run a Server on a Raspberry Pi, you will need to [compile from so
 1. Reload the systemd files `sudo systemctl daemon-reload` and restart the headless Server: `sudo systemctl restart jamulus-headless`
 1. Check all is well with `systemctl status jamulus-headless` (hit `q` to get back to the command prompt).
 
-### Configuration
-
 You can control Jamulus with the `systemctl` command. For example, to stop the Server cleanly:
 
 `sudo systemctl stop jamulus-headless`
 
+_To upgrade your Server, just repeat the steps above._
+
+### Configuration
```

Comment:
Still none of this (lines 98 to 130 new) is configuration.  Merge it with all the other running bits (including desktop).